admin/app/components/solidus_admin/main_nav_component.rb

```diff
@@ -3,5 +3,14 @@
 module SolidusAdmin
   # Renders the main navigation of Solidus Admin.
   class MainNavComponent < BaseComponent
+    erb_template <<~ERB
+      <nav>
+        <%=
+          render component("main_nav_item").with_collection(
+            container.all("main_nav").sort_by(&:position)
```

Comment:
What do you think about?

```rb
components.all("main_nav")...
```

as helper for the `container` implementation detail?

Reply:
See https://github.com/solidusio/solidus/pull/5099#discussion_r1207108938